blocks/columns/columns.js

```diff
@@ -141,9 +141,13 @@ export default function decorate(block) {
           anchor.classList.add('video-modal');
           anchor.appendChild(picture);
 
-          // remove empty paragraphs
+          // remove empty paragraphs but keeping any kind of immediate label for that Video intact
```

Comment:
I think instead of looking at the previous sibling, we can improve the empty paragraph check to additionally assert that the paragraph should only be removed if it has no text content as well.